### ISSUE TITLE
ROC section: bad example

### ISSUE THREAD

**Issue by berndbischl** [mlr-org/mlr-tutorial#18]
In the ROC adv section we compare some learners on sonar.task with a visual ROC curve.
But we train and test and the whole task, so we compare on the training set.

This is not common and a bad example. we SHOULD REALLY use at least a proper test set.

**Comment by schiffner** [comment]
Yep. I'm going to fix this.
